information-schema.md

```diff
@@ -0,0 +1,83 @@
+
+## Tables that are TiDB extensions
+
+| Table Name                                                                              | Description |
+|-----------------------------------------------------------------------------------------|-------------|
+| [`ANALYZE_STATUS`](/information-schema/information-schema-analyze-status.md)            | Provides information about tasks to collect statistics. |
+| [`CLUSTER_CONFIG`](/information-schema/information-schema-cluster-config.md)            | Provides details about configuration settings for the entire TiDB cluster. |
+| [`CLUSTER_HARDWARE`](/information-schema/information-schema-cluster-info.md)            | Provides details on the underlying physical hardware discovered on each TiDB component. |
+| [`CLUSTER_INFO`](/information-schema/information-schema-cluster-info.md)                | Provides details on the current cluster topology. |
+| [`CLUSTER_LOAD`](/information-schema/information-schema-cluster-load.md)                | Provides current load information for TiDB servers in the cluster. |
+| [`CLUSTER_LOG`](/information-schema/information-schema-cluster-log.md)                  | Provides a log for the entire TiDB cluster |
+| `CLUSTER_PROCESSLIST`                                                                   | Provides a cluster-level view of the `PROCESSLIST` table. |
+| `CLUSTER_SLOW_QUERY`                                                                    | Provides a cluster-level view of the `SLOW_QUERY` table. |
+| `CLUSTER_STATEMENTS_SUMMARY`                                                            | Provides a cluster-level view of the `STATEMENTS_SUMMARY` table. |
+| `CLUSTER_STATEMENTS_SUMMARY_HISTORY`                                                    | Provides a cluster-level view of the `CLUSTER_STATEMENTS_SUMMARY_HISTORY` table. |
+| [`CLUSTER_SYSTEMINFO`](/information-schema/information-schema-cluster-systeminfo.md)    | Provides details about kernel parameter configuration for servers in the cluster. |
+| [`DDL_JOBS`](/information-schema/information-schema-ddl-jobs.md)                        | Provides similar output to `ADMIN SHOW DDL JOBS` |
+| [`INSPECTION_RESULT`](/information-schema/information-schema-inspection-result.md)      | Triggers internal diagnostics checks. |
+| [`INSPECTION_RULES`](/information-schema/information-schema-inspection-rules.md)        | A list of internal diagnostic checks performed. |
+| [`INSPECTION_SUMMARY`](/information-schema/information-schema-inspection-summary.md)    | A summarized report of important monitoring metrics. |
+| [`METRICS_SUMMARY`](/information-schema/information-schema-metrics-summary.md)          | A summary of metrics extracted from Prometheus. |
+| `METRICS_SUMMARY_BY_LABEL`                                                              | See `METRICS_SUMAMARY` table. |
```

Comment:
Found a typo METRICS_SUMAMARY -> METRICS_SUMMARY. Can you help fix it? @nullnotnil

Reply:
Fixed it in https://github.com/pingcap/docs/pull/3470 (including some other fix)